test/cpp/ext/gsm/metadata_exchange_test.cc

```diff
@@ -123,12 +122,8 @@ TEST_F(MetadataExchangeTest, ClientAttemptStarted) {
 }
 
 TEST_F(MetadataExchangeTest, ClientAttemptDuration) {
-  Init(/*metric_names=*/{grpc::internal::
-                             OTelClientAttemptDurationInstrumentName()},
-       /*resource=*/TestGkeResource(),
-       /*labels_injector=*/
-       std::make_unique<grpc::internal::ServiceMeshLabelsInjector>(
-           TestGkeResource().GetAttributes()));
+  Init(/*metric_names=*/{
+      grpc::internal::OTelClientAttemptStartedInstrumentName()});
```

Comment:
Shouldn't this be `OTelClientAttemptDurationInstrumentName()`?

Reply:
Fixed